translator/config/schema.json

```diff
@@ -1178,6 +1196,118 @@
       },
       "additionalProperties": false
     },
+    "jmxObjectDefinition": {
+      "type": "object",
+      "properties": {
+        "endpoint": {
+          "description": "The endpoint to listen for JMX",
+          "$ref": "#/definitions/endpointOverrideDefinition"
+        },
+        "metrics_collection_interval": {
+          "$ref": "#/definitions/timeIntervalDefinition"
+        },
+        "username": {
+          "description": "The username to use for JMX authentication",
+          "type": "string"
+        },
+        "password_file": {
+          "description": "The password file to read in the passwords for JMX authentication",
+          "type": "string"
+        },
+        "keystore_path": {
+          "description": "The keystore path is required if SSL is enabled on the target JVM",
+          "type": "string"
+        },
+        "keystore_type": {
+          "description": "The keystore type if required by SSL",
+          "type": "string"
+        },
+        "truststore_path": {
+          "description": "The truststore path is required if SSL is enabled on the target JVM",
+          "type": "string"
+        },
+        "truststore_type": {
+          "description": "The truststore type if required by SSL",
+          "type": "string"
+        },
+        "registry_ssl_enabled": {
+          "description": "If SSL is enabled for RMI registry",
+          "type": "boolean"
+        },
+        "remote_profile": {
+          "description": "Supported JMX remote profiles in combination with SASL profiles",
+          "type": "string"
+        },
+        "realm": {
+          "description": "The realm, as required by remote profile SASL/DIGEST-MD5.",
+          "type": "string"
+        },
+        "jvm": {
```

Comment:
Shouldnt this be a `oneOf` these targets only?
What would happen if I specify multiple targets here?